libs/shared-utils/src/ErrorHandler.ts

```diff
@@ -0,0 +1,7 @@
+export let errorHandler: (error: Error) => void = (error: Error) => {
+    console.error(`${error.name} : ${error.message}`);
+};
+
+export const setErrorHandler = (newErrorHandler: (error: Error) => void) => {
+    errorHandler = newErrorHandler;
+};
```

Comment:
setErrorHandler will not replace error handler for errors declared BEFORE setErrorHandler was called. Which could cause weird issues.

You might want to try something like:

```suggestion
let activeErrorHandler: (error: Error) => void = (error: Error) => {
    console.error(`${error.name} : ${error.message}`);
};

export const errorHandler: (error: Error) => void = (error: Error) => {
    activeErrorHandler(error);
};

export const setErrorHandler = (newErrorHandler: (error: Error) => void) => {
    activeErrorHandler = newErrorHandler;
};
```